data/categories.txt

```diff
@@ -14495,6 +14891,7 @@ id:4Defibrilator
 it:4Defibrillatore
 ja:除細動器
 ko:제세동기
+lv:4Defibrilators
```

Comment:
also "Automatizētais ārējais defibrilators"

Reply:
It is unlikely that someone will type this in the search...